tasks/CentOS/main.yml

```diff
@@ -4,18 +4,18 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Set sensu_pkg_version {{ ansible_distribution }} override
```

Comment:
This is a bug in my code, I should be calling something like ```sensu_pkg_version: "{{ _sensu_pkg_version }}"```

I'll update this PR in just a bit